app/blueprints/round/templates/round_details.html

```diff
@@ -0,0 +1,551 @@
+{% from "govuk_frontend_jinja/macros/attributes.html" import govukAttributes -%}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+{% from "govuk_frontend_jinja/components/table/macro.html" import govukTable %}
+{%- from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary -%}
+{%- from "govuk_frontend_jinja/components/details/macro.html" import govukDetails -%}
+{% from "govuk_frontend_jinja/components/notification-banner/macro.html" import govukNotificationBanner %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{%- from "govuk_frontend_jinja/components/summary-list/macro.html" import govukSummaryList -%}
+{% set showNavigationBar = True %}
+{% set active_item_identifier = "applications" %}
+{% extends "base.html" %}
+{% block content %}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        {{ govukBackLink({
+          "text": "Back",
+          "href": url_for("round_bp.view_all_rounds")
+        }) }}
+    </div>
+</div>
```

Comment:
Looks like we've done this wrong everywhere at the moment, so let's leave this for now and we'll fix it later - but backlinks should be in the `beforeContent` block, not the main `content` block.